db/migrations/sqlite/000027_create_key_fields.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE batches ADD COLUMN "key" VARCHAR(1024);
+UPDATE batches SET "key" = "";
```

Comment:
There's no way to make this `NOT NULL` in sqlite due to the migration path, is there? 😞

Reply:
Yeah, that's I conclusion I came to.